### DISCUSSION TITLE
Want to translate document to chinese

### DISCUSSION THREAD

**Discussion by Gavin-Gong** [solidjs/solid#534] [General]
How do I start my work?
should I add `zh-cn` folder to `solid/documentation` or just create a new repo for chinese document?
any suggestiones? @ryansolid

**Comment by ryansolid** [comment]
@davedbase has been handling the site. I will tag him too.

**Comment by davedbase** [comment]
Ah, that would be wonderful. I didn't expect to get the request so soon but there are some discussions regarding translating the whole website if you might be able to support us on that as well? Specifically the tutorials which are a really important source of learning Solid. @Gavin-Gong as to not block you from going ahead, definitely create a folder just kindly maintain the same md and sub-folder structure.

@ryansolid with this request I think it's time we move all documentation to `solid-documentation`. It's fine leaving it where it is but we have to anticipate that there will be more copy change and localisation PRs coming in. Last we discussed this versioning was the hold back I believe?

**Reply by Gavin-Gong** [reply]
> I didn't expect to get the request so soon but there are some discussions regarding translating the whole website if you might be 
able to support us on that as well?

of course


> with this request I think it's time we move all documentation to solid-documentation

@davedbase Totally agree with you about moving documentation to `solid-documentation`.  

I will start my translation work base on this repo. Then we can relocate translation work later
